test.sh

```diff
@@ -18,7 +18,10 @@ export BRANCH_NAME=${BRANCH_NAME:-$(git symbolic-ref --short HEAD)}
 
 # Skip the integration tests if the Summon variables are not present
 if [ -z "$CF_API_ENDPOINT" ]; then
-    INTEGRATION_TAG="--tags ~@integration"
+    # Due to single quote requirements for multi-word tag arguments,
+    # this formatting is necessary
+    ARG="--tags"
+    INT="not @integration"
```

Comment:
should these be set to empty vars if the code doesn't hit this path?

Reply:
The shell will disregard them if they aren't set, which they will not be if we don't hit this path.
In any case, there were more issues with string interpolation, so I modified the command.